rodeo/src/rodeo/04-sequences.scala

```diff
@@ -3,37 +3,131 @@ package rodeo
 import zio.test._
 
 object Sequences extends Chapter {
+  /*
+    In this chapter we will study an important trait called Sequence. A sequence can have several different implementations
+    and the one we will use the most is List. In Scala a List is a Sequence that is implemented as an immutable linked list.
+    An important observation is that when we define a new sequence in Scala this sequence is automatically implemented as a List.
+    We can a have other different implementations of a Sequence, like a Vector or an Array but we will  use them only as
+    needed.
+
+  */
 
   /*
-  - head
-  - headOption
-  - concat
-  - pattern matching
-  -
-   */
-  Exercise(
-    "...."
-  ) {
+    In the following exercises we will introduce some of the most common methods defined for Sequences.
+    */
+
+
+  /*
+    The head method returns the first element in a given Sequence. The tail method returns the Sequence that remains
+    after deleting the first element.
+
```

Comment:
Note that `.head` is an *unsafe* operation for empty lists -since you cant obtain the first element of an empty sequence-. And as such, it should not be used in production code and `.headOption` is preferred.